src/components/Calculator.js

```diff
@@ -1,59 +1,52 @@
-import React from 'react';
+import React, { useState } from 'react';
 import calculate from '../logic/calculate';
 
-// eslint-disable-next-line react/prefer-stateless-function
-class Calculate extends React.Component {
-  constructor(props) {
-    super(props);
-    this.state = {};
-    this.handleButtonClick = this.handleButtonClick.bind(this);
-  }
+function Calculate() {
+  const [object, setObject] = useState({});
```

Comment:
- [ ] **[OPTIONAL]** It's a good practice to use `arrow` function syntax over function keyword. This will improve our code and make us to write code with ES6 standards